src/components/utils/useReducedMotion.stories.mdx

```diff
@@ -0,0 +1,42 @@
+import {ArgsTable, Meta, Story, Canvas} from '@storybook/addon-docs';
+import Accordion from '../accordion/Accordion';
+import AccordionItem from '../accordion/AccordionItem';
+
+<Meta title="Utilities/useReducedMotion" />
+
+# useReducedMotion
+
+`useReducedMotion` is a custom hook for detecting the users motion preference.
+
+## Import
+
+```jsx
+import {useReducedMotion} from 'brainly-style-guide';
+```
+
+## Usage
+
+```jsx
+const shouldReduceMotion = useReducedMotion()
+
+<Accordion reduceMotion={shouldReduceMotion}>
+    <AccordionItem title={'example'} id="accordion-item-1">
+      example body
+    </AccordionItem>
+  </Accordion>
+
+```
+
+## Example (using styleguide Accordion)
+
+To see in action on MAC open system preferences then accessibility and check "reduce motion" option
+
+![To see in action on MAC open system preferences then accessibility and check "reduce motion" option](/reduced-motion-setting.png)
```

Comment:
I think `/reduced-motion-setting.png` is not working on preview (sb is published to subfolder). Let's change it to `reduced-motion-setting.png`

Reply:
@coderitual  After change It doesn't work either (works locally)

Reply:
oh maybe it's something handled by mdx (so `/` is added always). Fortunately we can fix it separately.